src/prebid.js

```diff
@@ -293,23 +293,25 @@ $$PREBID_GLOBAL$$.renderAd = function (doc, id) {
       if (adObject) {
         //save winning bids
         $$PREBID_GLOBAL$$._winningBids.push(adObject);
+
         //emit 'bid won' event here
         events.emit(BID_WON, adObject);
 
-        var height = adObject.height;
-        var width = adObject.width;
-        var url = adObject.adUrl;
-        var ad = adObject.ad;
+        const { height, width, ad, mediaType } = adObject;
+        const url = adObject.adUrl;
 
-        if (doc === document || adObject.mediaType === 'video') {
+        if (doc === document) {
           utils.logError(`Error trying to write ad. Ad render call ad id ${id} was prevented from writing to the main document.`);
+        } else if (mediaType === 'video' || mediaType === 'video-outstream') {
```

Comment:
I don't think we want to render a pure `video` response yet, since that requires a video player